src/PrestaShopBundle/Resources/config/services/adapter/customer_service.yml

```diff
@@ -34,3 +34,24 @@ services:
     tags:
       - name: tactician.handler
         command: 'PrestaShop\PrestaShop\Core\Domain\CustomerService\Command\ReplyToCustomerThreadCommand'
+
+  prestashop.adapter.customer_service.command_handler.delete_customer_thread_handler:
```

Comment:
As suggested in a previous comment all those new handlers could be added to the core namespace instead
Maybe some others that you created recently could also be moved, but it should be handled in another PR